vividus-plugin-datetime/src/main/java/org/vividus/expression/FormatDateExpressionProcessor.java

```diff
@@ -48,11 +50,11 @@ public FormatDateExpressionProcessor(DateUtils dateUtils)
     @Override
     protected String evaluateExpression(Matcher expressionMatcher)
     {
-        ZonedDateTime zonedDate = dateUtils.parseDateTime(expressionMatcher.group(INPUT_DATE_GROUP),
-                ISO_STANDARD_FORMAT);
+        ZonedDateTime zonedDate = dateUtils.parseDateTime(
+                normalize(expressionMatcher.group(INPUT_DATE_GROUP)), ISO_STANDARD_FORMAT);
```

Comment:
does it make sense to introduce custom `ExpressionArgumentsMatcher` that will wrap Java matcher and act as a decorator, for example: 
```java
public String getArgument(int argumentIndex)
{
    return normalize(matcher.group(argumentIndex));
}
```
I have not checked whether it fits well for all cases

Reply:
for a nise implementation there is some restriction related to the fact that`Pattern` and `Matcher` are final classes, but lets try to do smth over available interfaces